### PR TITLE
Fix: #885 setted instagram in AboutFragment

### PR DESCRIPTION
**Pull request by Akshit6828**
Signed-off-by: Akshit Mangotra <akshitmangotra@gmail.com>

### CODE REVIEW

**Review by Endem10n — APPROVED** [review]

File: android/src/com/mapswithme/maps/settings/AboutFragment.java [hunk 1]
```diff
@@ -55,6 +55,7 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     setupItem(R.id.telegram, false, root);
     setupItem(R.id.facebook, false, root);
     setupItem(R.id.twitter, false, root);
+    setupItem(R.id.instagram,false, root);
```
Comment (biodranik) [inline android/src/com/mapswithme/maps/settings/AboutFragment.java:58]: Missing space before false :)

Comment (Akshit6828) [reply android/src/com/mapswithme/maps/settings/AboutFragment.java:58]: Ok, sir. I'll add space and revert it back.

**Comment by Akshit6828** [comment]
Hi sir @biodranik. I think I need to again commit using Git CLI. So should I revert this commit back?

**Comment by Akshit6828** [comment]
Hi, sir @Endem10n. I committed my last commit using Git CLI but still, my DCO check is not being passed. Should I revert this commit?

**Comment by Endem10n** [comment]
Already merged into master in another PR. Closing

**Comment by Akshit6828** [comment]
> Already merged into master in another PR. Closing

Thanks for merging the PR sir. Looking forward to contributing more with my skills. 😃

**Pull request closed without merging**